### DISCUSSION TITLE
Which variant of call signs do you prefer?

### DISCUSSION THREAD

**Discussion by freearhey** [iptv-org/community#154] [General]
For example, NBC 11 Minneapolis MN:

https://livevideo01.kare11.com/hls/live/2014544/newscasts/live.m3u8 

1. KARE-TV
2. KARE
3. KARE-DT1
4. KARE-DT
5. KARE-HD
6. KARE1

**Comment by what-a** [comment]
I've yet to see value in a suffix for any US channel's call sign. However, I do see value in a trailing number to indicate which sub-channel it is. So, my preference would be KARE1

...and I'd also prefer some naming indication that it's the newscast version of the channel (I know you were only asking about the call sign, but I couldn't resist).

**Comment by MapGuy11** [comment]
It should be 3 as the US. standard is the call sign then DT (Digital Television) 1 or whatever the subchannel is.
